docs/docsite/rst/os_guide/windows_ssh.rst

```diff
@@ -0,0 +1,202 @@
+
+.. code-block:: powershell
+
+    # Set default to powershell.exe
+    $shellParams = @{
+        Path         = 'HKLM:\SOFTWARE\OpenSSH'
+        Name         = 'DefaultShell'
+        Value        = 'C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe'
+        PropertyType = 'String'
+        Force        = $true
+    }
+    New-ItemProperty @shellParams
+
+    # Set default back to cmd.exe
+    Remove-ItemProperty -Path HKLM:\SOFTWARE\OpenSSH -Name DefaultShell
+
+The new default shell setting will apply to the next SSH connection, there is no need to restart the ``sshd`` service. You can also use Ansible to configure the default shell:
+
+.. code-block:: yaml
+
+    - name: set the default shell to PowerShell
+      ansible.windows.win_regedit:
+        path: HKLM:\SOFTWARE\OpenSSH
+        name: DefaultShell
+        data: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
+        type: string
+        state: present
+
+    - name: reset SSH connection after shell change
+      ansible.builtin.meta: reset_connection
+
+    - name: set the default shell to cmd
+      ansible.windows.win_regedit:
+        path: HKLM:\SOFTWARE\OpenSSH
+        name: DefaultShell
+        state: absent
+
+    - name: reset SSH connection after shell change
+      ansible.builtin.meta: reset_connection
+
+The ``meta: reset_connection`` is important to ensure the subsequent tasks will use the new default shell.
+
+
+Ansible Configuration
+---------------------
+
+To configure Ansible to use SSH for Windows hosts, you must set two connection variables:
+
+* set ``ansible_connection`` to ``ssh``
+* set ``ansible_shell_type`` to ``powershell`` or ``cmd``
+
+The ``ansible_shell_type`` variable should reflect the ``DefaultShell`` configured on the Windows host. Other SSH options as documented under the :ref:`ssh <ssh_connection>` can also be set for the Windows host.
+
+
+SSH Authentication
+------------------
+Win32-OpenSSH authentication with Windows is similar to SSH authentication on Unix/Linux hosts. While there are many authentication methods that can be used there are typically three used on Windows:
+
++----------+----------------+---------------------------+-----------------------+
+| Option   | Local Accounts | Active Directory Accounts | Credential Delegation |
++==========+================+===========================+=======================+
+| Key      | Yes            | Yes                       | No                    |
++----------+----------------+---------------------------+-----------------------+
+| GSSAPI   | No             | Yes                       | Yes                   |
++----------+----------------+---------------------------+-----------------------+
+| Password | Yes            | Yes                       | Yes                   |
++----------+----------------+---------------------------+-----------------------+
+
+* Key authentication
+* GSSAPI/Kerberos authentication
+* Password authentication
```

Comment:
This list seems redundant with the table above it.

Reply:
Thanks, will remove it